lib/utils/getInterestOptions.ts

```diff
@@ -23,5 +23,8 @@ export type InterestType = (typeof interests)[number];
 
 export const getInterestOptions = () => {
   // returning mutable type array instead of `Read Only` type.
```

Comment:
Since we're refactoring this, that comment does not line up with what's happening. It can be removed.